cloud-batch-python/python-batch/src/google_cloud_pybatch.egg-info/PKG-INFO

```diff
@@ -0,0 +1,76 @@
+Metadata-Version: 2.1
```

Comment:
we probably don't want to include the egg in the repo?

Reply:
Removed